src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -92,6 +93,23 @@
             'required': ['type', 'label', 'name'],
         },
 
+        'textarea': {
+            'type': 'object',
+            'properties': {
+                'type': {
+                    'type': 'string',
+                    'enum': ['textarea'],
+                },
+                'label': {
+                    'type': 'string',
+                },
+                'name': {
+                    'type': 'string',
+                },
+            },
+            'required': ['type', 'label', 'name'],
+        },
+
```

Comment:
We need to add `default` in here too and probably define it as being `oneOf(boolean, string)`. Same for `text`.

Reply:
I don't think we want to have a default for `text` or `textarea` to be a boolean do we?

Reply:
No, you're right. I was thinking of the `select` options, but that's something totally different.